### DISCUSSION TITLE
How exclude filetype?

### DISCUSSION THREAD

**Discussion by patsevanton** [phiresky/ripgrep-all#168] [Q&A] [answered]
Thanks for rga! 
How exclude search by filetype *.srt?

I search text, but i want exclude file with filetype srt.

**Comment by phiresky** [comment, marked as answer]
Most parameters work the same as in ripgrep. Just replace rg with rga. Quoting ChatGPT:


To exclude `.srt` files while using `ripgrep`, you need to use the `--glob` flag with the `!` operator which stands for "not". Here's an example command:

```bash
rg --glob '!*.srt' <pattern>
```

In the command above, replace `<pattern>` with the pattern you're searching for. This will search for your pattern in all files except those with the `.srt` extension.

You can add more file types to be excluded by adding more `--glob` parameters. For example, to exclude both `.srt` and `.txt` files, you can use:

```bash
rg --glob '!*.srt' --glob '!*.txt' <pattern>
```

Remember that the `!` operator negates the match, so `!*.srt` matches anything that is not an `.srt` file. The `--glob` flag tells `ripgrep` to apply a glob pattern to the files it searches.

**Comment by patsevanton** [comment]
Thanks!
